ingestion/functions/parsing/new_zealand/new_zealand_test.py

```diff
@@ -47,20 +47,23 @@
 
 
 def _c(location, gender, age_start, age_end, confirmed_date, travel=False, notes=None):
+    demographics = {}
+    if age_start is not None and age_end is not None:
+        demographics["ageRange"] = {"start": age_start, "end": age_end}
```

Comment:
is there value in keeping partial information? Can we create a range if we only have the start age, for example? (Also: does that ever happen?)

Reply:
it's always 90+ or X to Y, also this is for the test runner which should always have both limits specified